CL/cl_ext.h

```diff
@@ -2446,6 +2446,50 @@ typedef struct _cl_queue_family_properties_intel {
 ***************************************************************/
 #define cl_intel_sharing_format_query 1
 
+/***************************************************************
+* cl_ext_image_requirements_info
+***************************************************************/
+
+#ifdef CL_VERSION_3_0
```

Comment:
Mostly as an FYI, the header generation script will currently generate this condition slightly differently, and will guard any new types or enums or functions, but it will not guard the extension define itself.  Is that OK, or should I look into guarding entire extensions for situations like these?

For reference, the header generation PR is: https://github.com/KhronosGroup/OpenCL-Headers/pull/161

Reply:
IMHO : the extension define needs to be guarded as well. I can see the extension define being used to check if the extension is available for example ?